TeamCode/src/main/java/org/firstinspires/ftc/teamcode/OpModes/ArmTest.java

```diff
@@ -0,0 +1,50 @@
+package org.firstinspires.ftc.teamcode.OpModes;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.teamcode.Robot.Systems.Claw;
+import org.firstinspires.ftc.teamcode.Robot.Systems.SpampleArm;
+
+@TeleOp
+public class ArmTest extends LinearOpMode {
+    SpampleArm spampleArm;
+    @Override
+    public void runOpMode() throws InterruptedException {
+        spampleArm = new SpampleArm(hardwareMap);
+        waitForStart();
+        while (opModeIsActive()) {
+           
+            if (gamepad2.a) {
+                spampleArm.rotateWristTo(90);
```

Comment:
Add more comments to explain what these buttons do.

Reply:
we chose to not comment these because this was just for testing purposes and is not final or going to be used every again. hope that makes sense :)